garage/tf/core/mlp.py

```diff
@@ -46,8 +46,8 @@ def mlp(input_var,
                 inputs=l_hid,
                 units=hidden_size,
                 activation=hidden_nonlinearity,
-                kernel_initializer=hidden_w_init,
-                bias_initializer=hidden_b_init,
+                kernel_initializer=hidden_w_init(),
```

Comment:
why change from passing the instance to passing the type?

Reply:
we cannot pickle an instance, but we can pickle the type.

Reply:
what if i want to use `tf.constant_initializer(5)`?

Reply:
we pass a lambda

Reply:
is that pickleable?

Reply:
emm you are right. It's not.

Reply:
Can we use cloudpickle? It can pickle lambda.

Reply:
no, i think we should stick to the standard pickling protocol